ConnectorArchicad/ConnectorArchicad/Converters/ModelConverter.cs

```diff
@@ -31,6 +32,121 @@ public static List<Mesh> MeshesToSpeckle(MeshModel meshModel)
     }
 
     public static MeshModel MeshToNative(IEnumerable<Mesh> meshes)
+    {
+      var mergedEdges = new Dictionary<Tuple<Vertex, Vertex>, Tuple<List<int>, List<Tuple<int, int>>>>();
+
+      int polygonIndexCounter = 0;
+      var vertexOffset = 0;
+
+      var meshModel = new MeshModel();
+      var enumerable = meshes as Mesh[] ?? meshes.ToArray();
+      foreach (var mesh in enumerable)
+      {
+        meshModel.vertices.AddRange(mesh.GetPoints().Select(p => Utils.PointToNative(p)));
```

Comment:
Quick comment on performance, minor, but I'd thought I'd share my findings from other connectors.

In other connectors, we have optimised the performance of MeshToNative quite significantly by avoiding using `Point`s and reusing the scaling factor (calling `Units.GetConversionFactor` once per Mesh)

Creating a `Point` object for each vertex leads to a lot of heap allocations,
additionally, I see, `PointToNative` will call `Units.GetConversionFactor` **for each point**, which has a small cost, but one that adds up, especially when you consider meshes with high vert count.